CHANGES.rst

```diff
@@ -29,6 +29,7 @@ Unreleased
 
 * Removed the Context class in favour of simple storing the context as a dictionary.
 
+* Removed the deprecated ``zalando...`` annotations. This will require a 2.0 release.
```

Comment:
I think it would be more consistent to add new changes at the top? 🤔

Reply:
TBH I was always confused by this and would usually add them at the bottom. Need to remember!... Changed now, thanks.